### ISSUE TITLE
Transcript quantification

### ISSUE THREAD

**Issue by kvittingseerup** [ksahlin/isONcorrect#12]
Hi Kristoffer
(great name btw ;) )

Do you know if it is possible to make a reference-free transcript assembly from the output of isOncorrect?

Cheers
Kristoffer

**Comment by ksahlin** [comment]
Hi @kvittingseerup,

Indeed ;)

We are working on it :) Hope to get back to you soon about it.

What we have used up until now is the pipeline
`isONclust -> isONcorrect -> IsoCon`

where we have run `IsoCon` on each corrected-reads cluster from `isONcorrect`. We then set the parameter `--ignore_ends_len 30` to `IsoCon` (default 15) to account for larger differences in 3' and 5' ends that what `IsoCon` was designed for. `IsoCon` was originally designed for targeted data using start and end primers where the 3' and 5' sites were fairly fixed. 

It is definitely not a perfect solution both from an accuracy and a runtime aspect, in large part due to the variability in (ONT) data at 3' and 5' ends. The problem needs a tailored solution.

Best,
Kristoffer
